### PR TITLE
Added addnext command and an argument for listsongs command

### LINKED ISSUE #62: [Request] Add !addnow
Add a command (maybe !addnow) that allow an admin to bypass the queue and provide a song before all others peoples

### PR DESCRIPTION
**Pull request by nkhoit**
Made to resolve #62 
Added an admin command to add a song/playlist after the current song.
This has to funky interaction if you have a playlist within a playlist. (if the current song is part of a playlist, you addnext a playlist, then skip the current playlist, it'll skip the current song and the rest of the playlist after the addnext playlist. This may be the intent, but not too sure what it should do)

Also cleaned up that old comment about the playlist sizes in add.

### CODE REVIEW

**Comment by matthieugrieger** [comment]
Awesome!

Yeah, that is a bit of a weird case. Ideally when a playlist skip is executed it should only skip the playlist of the current track. I believe that the way the `!skipplaylist` command is coded up at the moment is what is making this not work properly.

I will take a look at this tomorrow.

**Comment by nkhoit** [comment]
I added a commit that add functionality to listsongs referenced in #104.

File: README.md [hunk 1]
```diff
@@ -44,9 +44,10 @@ These are all of the chat commands currently supported by MumbleDJ. All command
...
 **skip**| Submits a vote to skip the current song. Once the skip ratio target (specified in `mumbledj.gcfg`) is met, the song will be skipped and the next will start playing. Each user may only submit one skip per song. | None | No | `!skip`
-**skipplaylist** | Submits a vote to skip the current playlist. Once the skip ratio target (specified in mumbledj.gcfg) is met, the playlist will be skipped and the next song/playlist will start playing. Each user may only submit one skip per playlist. | None | No | `!skipplaylist`
+**skipplaylist** | Submits a vote to skip the current playlist. Once the skip ratio target (specified in `mumbledj.gcfgx`) is met, the playlist will be skipped and the next song/playlist will start playing. Each user may only submit one skip per playlist. | None | No | `!skipplaylist`
```
Comment (matthieugrieger) [inline README.md]: @nkhoit: Could you fix this typo?

**Comment by nkhoit** [comment]
fixed

**Comment by matthieugrieger** [comment]
Looks good, thanks again! :)

**Pull request merged**